### PR TITLE
General Update for 4.6 Status Tool

### PR DESCRIPTION
**Pull request by lsalazar1**
## Overview
Generic version updates and removal of certain components no longer in use for CP4AIOPS

### CODE REVIEW

**Comment by lsalazar1** [comment]
For sample outputs, please view the markdown file. Thanks!

File: kubectl-plugin/waiops-status.md [hunk 1]
```diff
@@ -22,8 +22,10 @@ Below are example outputs of these commands.
 
 ### Installation status checker output (`oc waiops status`)
 ```
-$ oc waiops status
-Cloud Pak for AIOps v4.4 installation status:
+liamsalazar@Liams-MBP kubectl-plugin % oc waiops status
```
Comment (griffindvs) [inline kubectl-plugin/waiops-status.md:25]: My want to clean this up a bit to `$ oc waiops status`

File: kubectl-plugin/waiops-status.md [hunk 2]
```diff
@@ -56,19 +58,21 @@ Cloud Pak for AIOps v4.4 installation status:
 
 ### Detailed installation status checker output (`oc waiops status-all`)
 ```
-$ oc waiops status-all
-Cloud Pak for AIOps v4.3 installation status:
+liamsalazar@Liams-MBP kubectl-plugin % oc waiops status-all
```
Comment (griffindvs) [inline kubectl-plugin/waiops-status.md:61]: Same note here

**Review by griffindvs — APPROVED** [review]
Minor note on docs

**Pull request merged**